Echo/EchoHelp.cpp

```diff
@@ -63,7 +78,7 @@ void EchoHelp::printHelp(const std::vector< std::string > & helpFlags ,
 
     cout << "Usage: \n $polysync-echo [options] \n\n";
 
-    for( auto index = 0; index < 4; ++index )
+    for( auto index = 0; index < 5; ++index )
```

Comment:
This seems like a magic number. Is there a way to get the number of help flags / descriptions ?